server/src/workspace/workspace-query-runner/workspace-query-runner.service.ts

```diff
@@ -127,6 +129,43 @@ export class WorkspaceQueryRunnerService {
     )?.records?.[0];
   }
 
+  async updateMany<Record extends IRecord = IRecord>(
+    args: UpdateManyResolverArgs<Record>,
+    options: WorkspaceQueryRunnerOptions,
+  ): Promise<Record | undefined> {
+    const { workspaceId, targetTableName } = options;
+
+    const query = this.workspaceQueryBuilderFactory.updateMany(args, options);
+
+    const result = await this.execute(query, workspaceId);
+
+    return this.parseResult<PGGraphQLMutation<Record>>(
+      result,
+      targetTableName,
+      'update',
+    )?.records[0];
```

Comment:
@gitstart-twenty It's a `many` query, so we should directly return records and not the first element of the array.

```suggestion
    )?.records;
```